projects/showcase/src/app/components/button/showcase-button-styles.component.html

```diff
@@ -0,0 +1,113 @@
+<div class="container-fluid">
```

Comment:
Shouldn't these buttons be changed into systelab-buttons as well?

Reply:
I don't think so, this are the default bootstrap buttons with the L&F based in our styles.